src/plugins/layout/stateprovider.ts

```diff
@@ -89,12 +89,20 @@ export function createElementLayoutState(element: Element): IElementLayoutState
 
   elementState.layout = null;
   if (rect) {
-    elementState.layout = {
-      x: Math.round(rect.left),
-      y: Math.round(rect.top),
-      width: Math.round(rect.width),
-      height: Math.round(rect.height)
-    };
+    if (styles["visibility"] !== "hidden") {
```

Comment:
Why do you not capture the visibility style value explicitly rather and send that in Clarity?

Reply:
Good point, now sending it as part of Clarity.